includes/wizards/class-subscriptions-wizard.php

```diff
@@ -45,19 +45,19 @@ public function enqueue_scripts_and_styles() {
 			return;
 		}
 
-		wp_enqueue_script( 
-			'newspack-subscriptions-wizard', 
-			Newspack::plugin_url() . '/assets/dist/subscriptions.js', 
-			[ 'wp-components' ], 
-			filemtime( dirname( NEWSPACK_PLUGIN_FILE ) . '/assets/dist/subscriptions.js' ), 
-			true 
+		wp_enqueue_script(
+			'newspack-subscriptions-wizard',
+			Newspack::plugin_url() . '/assets/dist/subscriptions.js',
+			[ 'wp-components' ],
+			filemtime( dirname( NEWSPACK_PLUGIN_FILE ) . '/assets/dist/subscriptions.js' ),
+			true
 		);
 
 		wp_register_style(
 			'newspack-subscriptions-wizard',
-			Newspack::plugin_url() . '/assets/dist/subscriptions-style.css',
-			[ 'newspack-components' ],
-			filemtime( dirname( NEWSPACK_PLUGIN_FILE ) . '/assets/dist/subscriptions-style.css' )
+			Newspack::plugin_url() . '/assets/dist/subscriptions.css',
+			null,
```

Comment:
```suggestion
			[ 'wp-components' ],
```

The stylesheets have a dependency on the `wp-components` stylesheet (see the code you removed in `class-wizard.php` ). That stylesheet provides the base style layer, and we just tweak some colors and stuff to make things look Muriel-designed. 

For example, we've lost a lot of tooltip styling now:
<img width="571" alt="Screen Shot 2019-05-06 at 2 28 40 PM" src="https://user-images.githubusercontent.com/7317227/57256849-9d26e080-700c-11e9-870a-dad4fc186e16.png">

Reply:
Oops sorry about that - I didn't study closely enough. Added the dependency back in https://github.com/Automattic/newspack-plugin/pull/38/commits/0dce134ad698022e46a0377a05f0c554aeffa723. Does that work, or do you think it should be in the `Wizard` class instead?